scripts/lite/configuration/shelley-3.yaml

```diff
@@ -286,6 +286,4 @@ TraceOptionLimiter:
     limiterName: ResourceLimiter
     limiterFrequency: 1.0
 TraceOptionForwarder:
-    address:
```

Comment:
So the socket configuration is decided with the `cardano-node` executable but the mode is decided with the yaml file?

Reply:
@Jimbo4350, thanks for catching this -- this was actually intended to both be decided by CLI -- fixed.

Reply:
So shouldn't the `TraceOptionForwarder:` be completely removed? But actually the actual current pattern is:
```
case makeNodeConfiguration $ defaultPartialNodeConfiguration <> configYamlPc <> cmdPc of
            Left err -> panic $ "Error in creating the NodeConfiguration: " <> Text.pack err
            Right nc' -> return nc'
```
The cli takes precedence but it should be possible to specify via the config yaml file.

Reply:
@Jimbo4350, problem is `TraceOptionForwarder` has other fields -- which are normally not changed in the config files, which is why you don't see them:

https://github.com/input-output-hk/cardano-node/blob/issue-3533-node-CLI-socket/trace-dispatcher/src/Cardano/Logging/Types.hs#L325-L331

Reply:
Alright fair enough, let's leave it as you prefer it